src/main/java/s8u/studies/myapplication/model/Pokedex_entries.kt

```diff
@@ -0,0 +1,10 @@
+package s8u.studies.myapplication.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Pokedex_entries (
+    @SerializedName("entry_number")
+    var id:Int,
```

Comment:
Esses valores com var, eles são atualizados em algum outro lugar? 

Se não, o que acham de trocar var por val? 

Assim mantemos o conceito de imutabilidade do kotlin, sempre começe com val e só mude pra var quando for altamente necessário.

https://medium.com/@regmoraes/programa%C3%A7%C3%A3o-funcional-em-kotlin-parte-1-dd63fbd42a6e